pkg/apis/workload/v1alpha1/types.go

```diff
@@ -155,4 +155,7 @@ const (
 	// InternalSyncTargetKeyLabel is an internal label set on a SyncTarget resource that contains the full hash of the SyncTargetKey, generated with the ToSyncTargetKey(..)
 	// helper func, this label is used for reverse lookups of a syncTargetKey to SyncTarget.
 	InternalSyncTargetKeyLabel = "internal.workload.kcp.dev/key"
+
+	// ComputeAPIBindingLabelKey is a annotation key to identify the APIBinding is for compute.
+	ComputeAPIBindingLabelKey = "internal.workload.kcp.dev/compute"
```

Comment:
```suggestion
	ComputeAPIBindingLabel = "internal.workload.kcp.dev/compute"
```
In order to follow the naming pattern of `InternalSyncTargetKeyLabel`